libs/coin-modules/coin-aptos/src/network/client.ts

```diff
@@ -202,6 +202,46 @@ export class AptosAPI {
     }
   }
 
+  async estimateFees(
+    transactionIntent: TransactionIntent<AptosAsset, AptosExtra, AptosSender>,
```

Comment:
Using types unrelated to the Alpaca API (base types or defined in the coin-aptos module) would make the AptosAPI class more resilient to external changes. What do you think?